more/release-notes.mdx

```diff
@@ -17,10 +17,14 @@ None
 
 ### New features
 
-* We'll be debuting in the coming week with self-service distribution of the Literal AI platform for self-hosting
+* **A/B test** feature is now available! A/B tests allow you to compare between generations.
+* We'll be debuting in the coming week with **self-service distribution** of the Literal AI platform for self-hosting
+* A new **user role** has been added: "Annotator", a user that can add tags and scores to the observability entities (e.g. thread, step...) and has no access outside of those.
+* Project administrator can now pick the role of a user when inviting.
 
 ### Improvements
 
+* Literal AI API keys are now shared among the team administrators (was only for a user).
```

Comment:
Sorry I was not clear, before, admins could create "personal" api keys, now they are shared in the project.
We keep the behavior of restricting access to admins.